styles.css

```diff
@@ -0,0 +1,3 @@
+.HDColour{
+    color:rgb(7, 100, 177)
+}
```

Comment:
- [ ] Your CSS is not well-formatted which is why I believe your styling linters are failing. 
A space is required before `{`. 
A space is required after the column `:` 
You need to terminate line 2 with a semi column `;` 
You also need to give a newline at the end of line 3

Please see the example below as a guide:
```
.HDColour {
    color: rgb(7, 100, 177);
} 

```

If you install styling, you can run `npx stylelint "**/*.{css,scss}"` to test locally if you linters are okay. 
Try as possible to make sure that all your linters passes.